# Awesome-Deep-Learning-in-Spatial-Transcriptomics
Awesome Deep Learning Works in Spatial Transcriptomics

If you have any problems, suggestions or improvements, please submit the issue or PR.

## Contents
* [Datasets](#datasets)
* [Papers](#papers)

## Datasets
Please refer to [this page]().

## Papers
Considering the increasing number of papers in this field, we roughly summarize some articles and put them into the following categories (they are still listed in this document):


- <a name="BLEEP"></a>**[BLEEP]** Spatially Resolved Gene Expression Prediction from H&E Histology Images via Bi-modal Contrastive Learning (**Arxiv**) [[paper](https://arxiv.org/pdf/2306.01859.pdf)] [[code](https://github.com/bowang-lab/BLEEP)]![GitHub stars](https://img.shields.io/github/stars/bowang-lab/BLEEP.svg?logo=github&label=Stars)

- <a name="Hist2ST"></a>**[Hist2ST]** Spatial transcriptomics prediction from histology jointly through Transformer and graph neural networks (**Briefings in Bioinformatics**) [[paper](https://academic.oup.com/bib/article-abstract/23/5/bbac297/6645485?redirectedFrom=fulltext&login=false)] [[code](https://github.com/biomed-AI/Hist2ST)]![GitHub stars](https://img.shields.io/github/stars/biomed-AI/Hist2ST.svg?logo=github&label=Stars)


- <a name="HisToGene"></a>**[HisToGene]** Leveraging information in spatial transcriptomics to predict super-resolution gene expression from histology images in tumors (**Biorxiv**) [[paper](https://www.biorxiv.org/content/10.1101/2021.11.28.470212v1.full.pdf)] [[code](https://github.com/maxpmx/HisToGene)]![GitHub stars](https://img.shields.io/github/stars/maxpmx/HisToGene.svg?logo=github&label=Stars)


- <a name="ST-Net"></a>**[ST-Net]** Integrating spatial gene expression and breast tumour morphology via deep learning (**Nature biomedical engineering**) [[paper](https://www.nature.com/articles/s41551-020-0578-x)] [[code](https://github.com/bryanhe/ST-Net)]![GitHub stars](https://img.shields.io/github/stars/bryanhe/ST-Net.svg?logo=github&label=Stars)


