# Awesome-Gene-Expression-Prediction-Methods-based-Histology-Images
Awesome Gene Expression Prediction Methods based Histology Images

If you have any problems, suggestions or improvements, please submit the issue or PR.

# Contents
* [Papers](##papers)
* [Datasets](##datasets)


## Papers

### 2024
- <a name="HGGEP"></a>**[HGGEP]** Gene Expression Prediction from Histology Images via Hypergraph Neural Networks (**Arxiv**) [[paper](https://github.com/QSong-github/HGGEP)] [[code](https://github.com/QSong-github/HGGEP)]![GitHub stars](https://img.shields.io/github/stars/QSong-github/HGGEP.svg?logo=github&label=Stars)

<!-- 
- <a name="iStar"></a>**[iStar]** Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology (**Cell Reports Medicine**) [[paper](https://www.nature.com/articles/s41587-023-02019-9)] [[code](https://github.com/daviddaiweizhang/istar)]![GitHub stars](https://img.shields.io/github/stars/daviddaiweizhang/istar.svg?logo=github&label=Stars)

- <a name="IGI-DL"></a>**[IGI-DL]** Harnessing TME depicted by histological images to improve cancer prognosis through a deep learning system (**Nature Biotechnology**) [[paper](https://pubmed.ncbi.nlm.nih.gov/38697103/)] [[code](https://github.com/ruitian-olivia/IGI-DL)]![GitHub stars](https://img.shields.io/github/stars/ruitian-olivia/IGI-DL.svg?logo=github&label=Stars)


-->



### 2023

- <a name="BLEEP"></a>**[BLEEP]** Spatially Resolved Gene Expression Prediction from H&E Histology Images via Bi-modal Contrastive Learning (**NIPS 2023**) [[paper]([https://arxiv.org/pdf/2306.01859.pdf](https://academic.oup.com/nar/article/50/22/e131/6761985?login=false))] [[code](https://github.com/bowang-lab/BLEEP)]![GitHub stars](https://img.shields.io/github/stars/bowang-lab/BLEEP.svg?logo=github&label=Stars)

- <a name="DeepPT"></a>**[DeepPT]** Prediction of cancer treatment response from histopathology images through imputed transcriptomics (**Journal of Clinical Oncology**) [[paper](https://ascopubs.org/doi/10.1200/JCO.2023.41.16_suppl.1551)] [[code](https://github.com/PangeaResearch/enlight-deeppt-data)]![GitHub stars](https://img.shields.io/github/stars/PangeaResearch/enlight-deeppt-data.svg?logo=github&label=Stars)

- <a name="SEPAL"></a>**[SEPAL]** SEPAL: Spatial Gene Expression Prediction from Local Graphs (**ICCV 2023**) [[paper](https://openaccess.thecvf.com/content/ICCV2023W/CVAMD/html/Mejia_SEPAL_Spatial_Gene_Expression_Prediction_from_Local_Graphs_ICCVW_2023_paper.html)] [[code](https://github.com/BCV-Uniandes/SEPAL)]![GitHub stars](https://img.shields.io/github/stars/BCV-Uniandes/SEPAL.svg?logo=github&label=Stars)

### 2022 and before

- <a name="Hist2ST"></a>**[Hist2ST]** Spatial transcriptomics prediction from histology jointly through Transformer and graph neural networks (**Briefings in Bioinformatics**) [[paper](https://academic.oup.com/bib/article-abstract/23/5/bbac297/6645485?redirectedFrom=fulltext&login=false)] [[code](https://github.com/biomed-AI/Hist2ST)]![GitHub stars](https://img.shields.io/github/stars/biomed-AI/Hist2ST.svg?logo=github&label=Stars)

- <a name="HisToGene"></a>**[HisToGene]** Leveraging information in spatial transcriptomics to predict super-resolution gene expression from histology images in tumors (**Biorxiv**) [[paper](https://www.biorxiv.org/content/10.1101/2021.11.28.470212v1.full.pdf)] [[code](https://github.com/maxpmx/HisToGene)]![GitHub stars](https://img.shields.io/github/stars/maxpmx/HisToGene.svg?logo=github&label=Stars)

- <a name="ST-Net"></a>**[ST-Net]** Integrating spatial gene expression and breast tumour morphology via deep learning (**Nature biomedical engineering**) [[paper](https://www.nature.com/articles/s41551-020-0578-x)] [[code](https://github.com/bryanhe/ST-Net)]![GitHub stars](https://img.shields.io/github/stars/bryanhe/ST-Net.svg?logo=github&label=Stars)


## Datasets

 -  human HER2-positive breast tumor ST data https://github.com/almaan/her2st/.
 -  human cutaneous squamous cell carcinoma 10x Visium data (GSE144240).
