# Awesome-Deep-Learning-in-Spatial-Transcriptomics
Awesome Deep Learning Methods in Spatial Transcriptomics

If you have any problems, suggestions or improvements, please submit the issue or PR.

## Contents
* [Datasets](#datasets)
* [Papers](#papers)

## Datasets
Please refer to [this page]().

## Papers about Image2Gene
Considering the increasing number of papers in this field, we roughly summarize some articles and put them into the following categories (they are still listed in this document):

- <a name="BLEEP"></a>**[BLEEP]** Spatially Resolved Gene Expression Prediction from H&E Histology Images via Bi-modal Contrastive Learning (**Arxiv**) [[paper]([https://arxiv.org/pdf/2306.01859.pdf](https://academic.oup.com/nar/article/50/22/e131/6761985?login=false))] [[code](https://github.com/bowang-lab/BLEEP)]![GitHub stars](https://img.shields.io/github/stars/bowang-lab/BLEEP.svg?logo=github&label=Stars)

- <a name="HGGEP"></a>**[HGGEP]** Gene Expression Prediction from Histology Images via Hypergraph Neural Networks (**Arxiv**) [[paper](https://github.com/QSong-github/HGGEP)] [[code](https://github.com/QSong-github/HGGEP)]![GitHub stars](https://img.shields.io/github/stars/QSong-github/HGGEP.svg?logo=github&label=Stars)

- <a name="Hist2ST"></a>**[Hist2ST]** Spatial transcriptomics prediction from histology jointly through Transformer and graph neural networks (**Briefings in Bioinformatics**) [[paper](https://academic.oup.com/bib/article-abstract/23/5/bbac297/6645485?redirectedFrom=fulltext&login=false)] [[code](https://github.com/biomed-AI/Hist2ST)]![GitHub stars](https://img.shields.io/github/stars/biomed-AI/Hist2ST.svg?logo=github&label=Stars)

- <a name="HisToGene"></a>**[HisToGene]** Leveraging information in spatial transcriptomics to predict super-resolution gene expression from histology images in tumors (**Biorxiv**) [[paper](https://www.biorxiv.org/content/10.1101/2021.11.28.470212v1.full.pdf)] [[code](https://github.com/maxpmx/HisToGene)]![GitHub stars](https://img.shields.io/github/stars/maxpmx/HisToGene.svg?logo=github&label=Stars)

- <a name="ST-Net"></a>**[ST-Net]** Integrating spatial gene expression and breast tumour morphology via deep learning (**Nature biomedical engineering**) [[paper](https://www.nature.com/articles/s41551-020-0578-x)] [[code](https://github.com/bryanhe/ST-Net)]![GitHub stars](https://img.shields.io/github/stars/bryanhe/ST-Net.svg?logo=github&label=Stars)


## Papers about Single-cell Spatial Transcriptomics

- <a name="SpaGE"></a>**[SpaGE]** SpaGE: Spatial Gene Enhancement using scRNA-seq (**Nucleic Acids Research**) [[paper](https://academic.oup.com/nar/article/48/18/e107/5909530?login=false)] [[code](https://github.com/tabdelaal/SpaGE)]![GitHub stars](https://img.shields.io/github/stars/tabdelaal/SpaGE.svg?logo=github&label=Stars)

- <a name="Tangram"></a>**[Tangram]** Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram (**Nature Methods**) [[paper](https://www.nature.com/articles/s41592-021-01264-7)] [[code](https://github.com/broadinstitute/Tangram)]![GitHub stars](https://img.shields.io/github/stars/broadinstitute/Tangram.svg?logo=github&label=Stars)

- <a name="Benchmarking"></a>**[Benchmarking]** Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution (**Nature Methods**) [[paper](https://www.nature.com/articles/s41592-022-01480-9)] [[code](https://github.com/QuKunLab/SpatialBenchmarking)]![GitHub stars](https://img.shields.io/github/stars/QuKunLab/SpatialBenchmarking.svg?logo=github&label=Stars)

- <a name="CytoSPACE"></a>**[CytoSPACE]** High-resolution alignment of single-cell and spatial transcriptomes with CytoSPACE (**Nature Biotechnology**) [[paper](https://www.nature.com/articles/s41587-023-01697-9)] [[code](https://github.com/digitalcytometry/cytospace)]![GitHub stars](https://img.shields.io/github/stars/digitalcytometry/cytospace.svg?logo=github&label=Stars)

- <a name="SpatialScope"></a>**[SpatialScope]** Integrating spatial and single-cell transcriptomics data using deep generative models with SpatialScope (**Nature Communications**) [[paper](https://www.nature.com/articles/s41467-023-43629-w)] [[code](https://github.com/YangLabHKUST/SpatialScope)]![GitHub stars](https://img.shields.io/github/stars/YangLabHKUST/SpatialScope.svg?logo=github&label=Stars)


